framework/tests/viewpointtest/PolyConstructor.java

```diff
@@ -3,9 +3,10 @@
 public class PolyConstructor {
 
     static class MyClass {
-        @SuppressWarnings({"inconsistent.constructor.type", "super.invocation.invalid"})
+        // :: error: (super.invocation.invalid) :: warning: (inconsistent.constructor.type)
         @PolyVP MyClass(@PolyVP Object o) {
-            throw new RuntimeException(" * You are filled with DETERMINATION."); // stub
+            // :: warning: (cast.unsafe.constructor.invocation)
+            throw new @A RuntimeException(" * You are filled with DETERMINATION."); // stub
```

Comment:
What's the point of this change? Should there be multiple versions?

Reply:
I expand the test suite in here.